react-components/src/components/Widgets/WindowWidget.tsx

```diff
@@ -140,6 +167,8 @@ const useResize = (
       const height = isMinimized ? WIDGET_WINDOW_MIN_HEIGHT : parentHeight * WIDGET_HEIGHT_FACTOR;
 
       setSize({ width, height });
+
+      onResize?.();
```

Comment:
Should this take the new width and height as arguments? In case the user wants to use those?